OOVPADatabase/Xapi.OOVPA.inl

```diff
@@ -127,57 +127,57 @@
 
-    REGISTER_OOVPAS(CreateMutex, 3911), // Too High Level (from 3911's comment)
-    REGISTER_OOVPAS(CreateThread, 3911), // Too High Level (from 3911's comment)
-    REGISTER_OOVPAS(ExitThread, 3911),//
-    REGISTER_OOVPAS(MoveFileA, 3911),
-    REGISTER_OOVPAS(SwitchToThread, 3911),
-    REGISTER_OOVPAS(XCalculateSignatureBegin, 3911, 4039),
-    REGISTER_OOVPAS(XapiBootDash, 3911), // obsolete (from 4721's comment)
-    REGISTER_OOVPAS(XapiInitProcess, 3911, 3950, 4242, 4831, 5028), // obsolete, Too High Level (from 4721's comment)
-    REGISTER_OOVPAS(XapiThreadStartup, 3911), // obsolete? (from 4627, 5028, 5558, 5788, 5849's comment) // obsolete (from 4721's comment)
-    REGISTER_OOVPAS(ConvertThreadToFiber, 3911),
-    REGISTER_OOVPAS(CreateFiber, 3911),
-    REGISTER_OOVPAS(DeleteFiber, 3911),
-    REGISTER_OOVPAS(GetExitCodeThread, 3911),
-    REGISTER_OOVPAS(GetOverlappedResult, 3911),
-    REGISTER_OOVPAS(GetThreadPriority, 3911),
-    REGISTER_OOVPAS(GetTimeZoneInformation, 3911),
-    REGISTER_OOVPAS(GetTypeInformation, 4134), // TODO: Actually introduced in some unknown XDK between 4134 and 4361
-    REGISTER_OOVPAS(OutputDebugStringA, 3911),
-    REGISTER_OOVPAS(QueueUserAPC, 3911),
-    REGISTER_OOVPAS(QueryPerformanceCounter, 3911),
-    REGISTER_OOVPAS(RaiseException, 3911),
-    REGISTER_OOVPAS(SetThreadPriority, 3911),
-    REGISTER_OOVPAS(SetThreadPriorityBoost, 3911),
-    REGISTER_OOVPAS(SignalObjectAndWait, 3911),
-    REGISTER_OOVPAS(SwitchToFiber, 3911),
-    REGISTER_OOVPAS(XAutoPowerDownResetTimer, 3911), // Just calls KeSetTimer (from 3911's comment)
-    REGISTER_OOVPAS(XFormatUtilityDrive, 4242),
-    REGISTER_OOVPAS(XGetDeviceChanges, 3911), // Was PATCH (from 5788's comment)
-    REGISTER_OOVPAS(XGetDeviceEnumerationStatus, 4831),
-    REGISTER_OOVPAS(XGetDevices, 3911),
-    REGISTER_OOVPAS(XGetLaunchInfo, 3911),
-    REGISTER_OOVPAS(XID_fCloseDevice, 3911, 4831),
-    REGISTER_OOVPAS(XInitDevices, 3911, 5120),
-    REGISTER_OOVPAS(XInputClose, 3911),
-    REGISTER_OOVPAS(XInputGetCapabilities, 3911, 4831),
-    REGISTER_OOVPAS(XInputGetDeviceDescription, 4831),
-    REGISTER_OOVPAS(XInputGetState, 3911, 4242, 5455),
-    REGISTER_OOVPAS(XInputOpen, 3911, 4242),
-    REGISTER_OOVPAS(XInputPoll, 3911),
-    REGISTER_OOVPAS(XInputSetState, 3911, 4242, 4831),
-    REGISTER_OOVPAS(XLaunchNewImageA, 3911, 4721, 5344),
-    REGISTER_OOVPAS(XMountAlternateTitleA, 3911, 5028, 5455),
-    REGISTER_OOVPAS(XMountMUA, 3911, 4242),
-    REGISTER_OOVPAS(XMountMURootA, 3911, 4242),
-    REGISTER_OOVPAS(XMountUtilityDrive, 3911, 4432),
-    REGISTER_OOVPAS(XRegisterThreadNotifyRoutine, 3911),
-    REGISTER_OOVPAS(XSetProcessQuantumLength, 4134),
-    REGISTER_OOVPAS(XUnmountAlternateTitleA, 3911),
-    REGISTER_OOVPAS(XapiFiberStartup, 3911),
-    REGISTER_OOVPAS(timeKillEvent, 3911),
-    REGISTER_OOVPAS(timeSetEvent, 3911),
+	REGISTER_OOVPAS(CreateMutex, 3911), // Too High Level (from 3911's comment)
```

Comment:
Please restore tab spaces

Reply:
Ugh, visual studio sucks sometimes, I didn't change anything regarding spacing, it auto-formatted like this when I typed :\

Reply:
We could avoid this by adding an editor configuration file to to root of all of our repositories

Reply:
@PatrickvL that quickly escalates to a collection of IDE configs and is generally not a good idea. Stricter review on this is usually enough - it doesn't take much time to check and fix.

There's also even a git filter in https://github.com/Cxbx-Reloaded/XbSymbolDatabase/blob/master/.gitconfig; however, it wasn't used here?

This repository already has an issue with being overly protective: https://github.com/Cxbx-Reloaded/XbSymbolDatabase/blob/master/.gitignore is massive bloat!

Normally such things shouldn't be handled in a projects .gitignore (which is a project-specific file).

For users who use such tools, git also allows a systemwide .gitignore which should be used to hide local files. Unfortunately many users (and maintainers) are unaware and this lead to projects having overly complicated project-files.
What if a new editor appears? Do we just keep adding to this list?

This should be similar for editor configurations: the project shouldn't have a catch-all.

It should just set and enforce the rules: tell users what they should do (and possibly provide simple, universally acceptable tools). For code-style, a single clang-format or checkpatch.pl can be provided, which have become defacto standard for code-formatting (independent of editors). But I don't think it's necessary here - review should be enough.

Reply:
@PatrickvL, I had talked to Luke about it and informed him there will be .clang-format to correct the problem.

@JayFoxRox It will be great to have a "whitelist" of what files *should* be included in the git instead of adding various IDEs to it.  Hmm... I found a possibility to do so for have whitelist in .gitignore file. Source: https://stackoverflow.com/questions/8024924/gitignore-ignore-all-files-then-recursively-allow-foo